test/mysql.test.js

```diff
@@ -878,7 +911,9 @@ describe('mysql', function() {
   after(function(done) {
     Post.destroyAll(function() {
       PostWithStringId.destroyAll(function() {
-        PostWithUniqueTitle.destroyAll(done);
+        PostWithUniqueTitle.destroyAll(function() {
+          Student.destroyAll(done);
```

Comment:
Ditto, use `async.each`? This function seems to be the same as the one above, can you extract it into a shared one please?